src/screens/ChatScreen/SignedChatScreen.tsx

```diff
@@ -24,7 +24,7 @@ const SignedChatScreen = () => {
     chats,
     goBackFromChatScreen,
     goToChatInfoScreen,
-    sendChat,
+    sendChatMutation,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `sendChat` to `sendChatMutation` seems appropriate as it aligns with the shift towards using mutation functions for sending chats. However, please ensure that error handling is properly implemented in `sendChatMutation` to handle any potential errors during the chat sending process.
```diff
-    sendChat,
+    sendChatMutation,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->